src/utils/registerCommands.ts

```diff
@@ -21,9 +21,9 @@ export async function registerCommands(
       method: "PUT",
       body: JSON.stringify(commands),
     });
-
-    if (response.ok) console.log("Registered all commands");
-    else console.error("Error Registering Commands");
+    if (response.ok) {
+      console.log("Registered all commands");
```

Comment:
why console

Reply:
When we run `npm run register` to register the command we built, we need an acknowledgement whether the command was registered successfully or not.